plugins/html-plugin/src/main/java/ru/sbtqa/tag/stepdefs/HtmlSetupSteps.java

```diff
@@ -6,11 +6,20 @@
 
 public class HtmlSetupSteps {
 
-    private HtmlSetupSteps() {}
+    static final ThreadLocal<WebDriverService> storage = new ThreadLocal<WebDriverService>() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this anonymous class with a call to "ThreadLocal.withInitial". ([squid:S4065](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS4065))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHOCkcrQ7oSxgxnf8&open=AWeIHOCkcrQ7oSxgxnf8)

[ISSUE_KEY]: AWeIHOCkcrQ7oSxgxnf8